src/runtime/config/configuration-acrn.toml.in

```diff
@@ -185,6 +185,9 @@ internetworking_model="@DEFNETWORKMODEL_ACRN@"
 # (default: true)
 disable_guest_seccomp=@DEFDISABLEGUESTSECCOMP@
 
+# disable applying SELinux on the VMM process (default false)
+disable_selinux=@DEFDISABLESELINUX@
```

Comment:
we may need an annotation to make this pod specific

Reply:
No. I don’t want a person to be able to specify. If an admin wants to provide this it should be through different runtimeclasses imo

Reply:
Yep, this is better to not be exposed via annotation.

Reply:
I'd add a huge warning here saying it's not advised to do so.
@tanweernoor, @egernst, something towards the lines off:
```
SELinux provides an extra security layer and disabling it may lead to security issues.
```

WDYT?

Reply:
imo its subjective, and I'm okay with as is, but I want other folks to weigh in